spec/book_spec.rb

```diff
@@ -0,0 +1,9 @@
+require_relative '../book'
+
+RSpec.describe Book do
+  it 'creates a new book' do
+    book = Book.new('Title', 'Author')
+    expect(book.title).to eq('Title')
+    expect(book.author).to eq('Author')
+  end
+end
```

Comment:
- [ ] Kindly make sure to add a test for the `add_rental` method inside the `Book` class to ensure it works correctly you can check the `rentals` instance variable of the book after adding a rental.